lib/src/room.dart

```diff
@@ -1648,102 +1650,171 @@ class Room {
     }
   }
 
-  final Set<String> _requestingMatrixIds = {};
-
-  /// Requests a missing [User] for this room. Important for clients using
-  /// lazy loading. If the user can't be found this method tries to fetch
-  /// the displayname and avatar from the profile if [requestProfile] is true.
-  Future<User?> requestUser(
+  // Internal helper to implement requestUser
+  Future<User?> _requestSingleParticipantViaState(
```

Comment:
_requestParticipantStateEventFromServer ?

Reply:
All of our "request" functions imply server requests, so that seems redundant? It also is an internal helper function, so the name doesn't seem that important.